common/src/main/java/org/mvndaemon/mvnd/common/Environment.java

```diff
@@ -252,11 +252,12 @@
     MVND_SOCKET_FAMILY("mvnd.socketFamily", null, "inet", OptionType.STRING, Flags.DISCRIMINATING),
     /**
      * Pattern that will force eviction of the plugin realms if one of its dependencies matches.
-     * The overall pattern is a comma separated list of:<ul>
-     *   <li>a glob pattern starting with <code>glob:</code> (the default syntax if no scheme is specified)</li>
-     *   <li>a regex pattern starting with <code>regex:</code></li>
-     *   <li>a maven expression, either <code>mvn:[groupId]:[artifactId]:[version]</code>,
-     *      <code>mvn:[groupId]:[artifactId]</code> or <code>mvn:[artifactId]</code></li>
+     * The overall pattern is a comma separated list of:
+     * <ul>
+     * <li>a glob pattern starting with <code>glob:</code> (the default syntax if no scheme is specified)</li>
+     * <li>a regex pattern starting with <code>regex:</code></li>
+     * <li>a maven expression, either <code>mvn:[groupId]:[artifactId]:[version]</code>,
+     * <code>mvn:[groupId]:[artifactId]</code> or <code>mvn:[artifactId]</code></li>
      * </ul>
```

Comment:
Does the HTML and JavaDoc `{@literal /}` formatting look good in `mvnd --help`?

Reply:
It should now:
```
-Dmvnd.pluginRealmEvictPattern=<string> Pattern that will force eviction of the plugin realms if one of its dependencies matches. The overall pattern is a
                                        comma separated list of either: a glob pattern starting with 'glob:' (the default syntax if no scheme is
                                        specified), a regex pattern starting with 'regex:', a maven expression, either
                                        'mvn:[groupId]:[artifactId]:[version]', 'mvn:[groupId]:[artifactId]' or 'mvn:[artifactId]'. This pattern will be
                                        evaluated against the full path of the dependencies, so it is usually desirable to start with 'glob:**/' to
                                        support any location of the local repository.
```

Reply:
Thanks, looks good!

I wonder whether a path against the pattern is matched can also be outside of the local maven repo? - Some sort of `$PWD/target/my.jar` is not possible?

Is this only matched against jar paths? Is there no chance that class directories are a part some classloader?

Reply:
For plugins, artifacts are always resolved and thus come from the local repository, so I don't think they can include things like `$PWD/target/my.jar` or `target/classes`.